_posts/2025-10-06-phoonnx.md

```
@@ -0,0 +1,125 @@
+
+### **Get Involved and Find the Models**
+
+We invite the community to explore and utilize these new resources. Your feedback is crucial to improving voice quality and expanding language coverage.
+
+| Resource                 | Description                                                             | Link                                                                                                             |
+|:-------------------------|:------------------------------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------|
+| **Phoonnx Models**       | The new phoonnx-trained TTS models in ONNX format.                      | [phoonnx-tts-models](https://huggingface.co/collections/TigreGotico/phoonnx-tts-models-68cd76d5b485394d9b71032e) |
+| **Piper/Phoonnx Voices** | The full collection of OpenVoiceOS voices compatible Piper.             | [pipertts-voices](https://huggingface.co/collections/OpenVoiceOS/pipertts-voices-68594d08f08e6ec56eddf4eb)       |
+| **Open Datasets**        | Datasets used for training these voices, furthering open-data research. | [tts-datasets](https://huggingface.co/collections/TigreGotico/tts-datasets-68dd4156a1484d2cf7bcbd5f)             |
+
+---
```

markdownlint-cli2 (0.18.1)

Horizontal rule style
Expected: -----; Actual: ---

(MD035, hr-style)